README.md

```diff
@@ -140,3 +141,17 @@ Start deployment
 ```bash
 ./dobi.sh cpu01-edgefarm-minio-deploy
 ```
+
+### Install Raspberry Pi sdimg on SD Card
+
+Find build images:
+
+```
+ls */install/images/raspberrypi4-64/core-image-base-raspberrypi4-64-*.sdimg
+```
+
+Install on SD Card:
+
+```
+sudo dd if=<PATH-TO-IMAGE>.sdimg of=<DEVICE> bs=1M && sudo sync
```

Comment:
Or on windows, use https://www.balena.io/etcher/

Add some notes about boot behaviour: 
- Kernel boot messages appear on serial console (is this a good idea?, dummy users would expect it on HDMI)
- ssh is enabled, locate device in network, look for  device named "raspberrypi4-64"
- login with ssh root@<ip>

Reply:
Added instructions how to install with Raspberry Pi Imager (Windows / Linux)
Login instructions not part of section `Install Images`